packages/components/input/src/textarea.tsx

```diff
@@ -140,7 +146,15 @@ const Textarea = forwardRef<"textarea", TextAreaProps>(
       <Component {...getBaseProps()}>
         {shouldLabelBeOutside ? labelContent : null}
         <div {...getInputWrapperProps()} data-has-multiple-rows={dataAttr(hasMultipleRows)}>
-          {shouldLabelBeInside ? labelContent : null}
+          {end ? (
+            <div className="flex pb-1 justify-between w-full items-center">
```

Comment:
don't define classes here. follow the pattern - use `getXXXXProps()`.

Reply:
OK